src/clients/rust/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "tigerbeetle"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+bitflags = "2.6.0"
+futures = "0.3.31"
+thiserror = "2.0.3"
```

Comment:
This is reasonable I guess! The 90% here would be to get rid of `thiserror` and implement the relevant errors manually, just to get rid of syn in transitive deps, but this would be a stretch goal.

Reply:
I made a PR to @brson's repo to remove thiserror dependency: https://github.com/brson/tigerbeetle/pull/1/files

Reply:
Merged! Thanks @austenadler